### ISSUE TITLE
Deprecation warning when precompiling assets after 4.2.0 -> 4.2.1 upgrade

### ISSUE THREAD

**Issue by steady286** [mastodon/mastodon#27550]
### Steps to reproduce the problem

1. Run a 4.2.0 (non-docker) installation
2.upgrade to 4.2.1 as specified in the release notes
```
$ git fetch && git checkout v4.2.1 
$bundle install
$ yarn install --frozen-lockfile
$ RAILS_ENV=production bundle exec rails assets:precompile
Compiling...
Compiled all packs in /home/mastodon/live/public/packs
`isModuleDeclaration` has been deprecated, please migrate to `isImportOrExportDeclaration`
    at isModuleDeclaration (/home/mastodon/live/node_modules/babel-plugin-lodash/node_modules/@babel/types/lib/validators/generated/index.js:2740:35)
    at PluginPass.Program (/home/mastodon/live/node_modules/babel-plugin-lodash/lib/index.js:102:44)
```

### Expected behaviour

Warning-free compile

### Actual behaviour

compilation yielded deprecation warnings

### Detailed description

no further text needed. Mastodon starts and is working.

### Mastodon instance

olching.social

### Mastodon version

V4.2.1

### Technical details

If this is happening on your own Mastodon server, please fill out those:
- Ruby version: 
ruby 3.2.2 (2023-03-30 revision e51014f9c0) [x86_64-linux]
- Node.js version: 
v16.19.0

**Comment by renchap** [comment]
This Babel plugin has not been updated in a long time, this deprecation notice can be safely ignored.

I plan to add a local patch for this package to remove it when we switch away from Yarn 1.

**Comment by renchap** [comment]
Fixed by #27073
